models/public/pelee-coco/pelee-coco.md

```diff
@@ -0,0 +1,81 @@
+# pelee-coco
+
+## Use Case and High-Level Description
+
+  The [Pelee](https://arxiv.org/pdf/1804.06882.pdf) is a Real-Time Object Detection System on Mobile Devices
+  based on Single Shot Detection approach. The model is is implemented using the
+  Caffe\* framework and trained on MSCOCO\* dataset.
+  For details about this model, check out the [repository](https://github.com/Robert-JunWang/Pelee).
```

Comment:
This whole block is unnecessarily indented.

Reply:
fixed